dev/com.ibm.ws.microprofile.rest.client_fat/publish/servers/mpRestClient10.headerPropagation/server.xml

```diff
@@ -4,6 +4,7 @@
     <feature>jaxrs-2.0</feature>
     <feature>mpRestClient-1.0</feature>
     <feature>appSecurity-2.0</feature>
+    <feature>servlet-4.0</feature>
```

Comment:
```suggestion
    <feature>servlet-3.1</feature>
```
jaxrs-2.0 is EE7.

Reply:
This is a mistake, but it will get overwritten by `MicroProfileActions.repeat()`. The test will never run with the exact configuration in the server.xml.

We should still fix it, but this is why it hasn't caused any failures.